src/routes/+layout.svelte

```diff
@@ -36,9 +36,11 @@
 	<nav>
 		<ul>
 			<li><a href="/">Home</a></li>
+			<li><a href="/signup">Signup</a></li>
 			<li><a href="/login">Login</a></li>
 			<li><a href="/reset">Reset</a></li>
 			<li><a href="/dashboard">Dashboard</a></li>
+			<li style="height: 100px;"><logout-button></logout-button></li>
```

Comment:
What's the behavior of the logout button when the user is not logged in?

Would it be worth conditionally rendering the button inside a [Svelte {#if} block](https://svelte.dev/docs/logic-blocks#if)?

Reply:
The Toolkit logout button displays whether or not a user is logged in, so good call on conditionally rendering. I removed it from the example to keep it simple for now. (I'm considering whether a "log in/log out" button, where the appearance and action changes based on whether the user is logged in, would make sense as a pre-built component for the Toolkit.)